app/styles/base.css

```diff
@@ -771,9 +771,16 @@ input[type=button]:active, select:active {
   flex-direction: column;
 }
 :root.noVNC_connecting #noVNC_transition,
-:root.noVNC_disconnecting #noVNC_transition {
+:root.noVNC_disconnecting #noVNC_transition,
+:root.noVNC_reconnecting #noVNC_transition {
   display: flex;
 }
+#noVNC_cancel_reconnect_button {
+  display: none;
+}
+:root.noVNC_reconnecting #noVNC_cancel_reconnect_button {
+  display: block;
+}
```

Comment:
Minor tip, here you can use the `:not` selector to avoid hard coding it's normal `display' state. There are some examples in that css file.